internal/http/services/owncloud/ocdav/propfind/propfind.go

```diff
@@ -458,7 +458,7 @@ func (p *Handler) getResourceInfos(ctx context.Context, w http.ResponseWriter, r
 					// add path to resource
 					res.Infos[i].Path = filepath.Join(info.Path, res.Infos[i].Path)
 					if spacesPropfind {
-						res.Infos[i].Path = utils.MakeRelativePath(res.Infos[i].Path)
+						res.Infos[i].Path = utils.MakeRelativePath(filepath.Join(info.Path, res.Infos[i].Path))
```

Comment:
Isn't this reintroducing a bug?
The `Join` is already happening in line 459.

Reply:
Good catch :+1: Removed it.